### PR TITLE
Transform package from template into something more workable

### PR DESCRIPTION
**Pull request by nikiburggraf**
Includes a few chores and setting up the basic skeletons of a streamlit app

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -28,25 +28,3 @@ jobs:
           fetch-depth: 0
 
       - uses: hynek/build-and-inspect-python-package@v2
-
-  publish:
```
Comment (lsetiawan) [inline .github/workflows/cd.yml:32]: Let's bring this back, I think we can publish to pypi in the future!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+```bash
+pre-commit install # Will install a pre-commit hook into the git repo
+```
+
+NOTE: You can also/alternatively run `pre-commit run` (changes only) or
+`pre-commit run --all-files` to check even without installing the hook.
+
+## Run Neglected Diagnostics
+
+You are now ready to run the Neglected Diagnostics Streamlit application:
```
Comment (lsetiawan) [inline CONTRIBUTING.md:52]: Mentioned of ND is still here, modify to current project

File: pyproject.toml [hunk 3]
```diff
@@ -21,16 +21,19 @@ classifiers = [
...
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
   "Topic :: Scientific/Engineering",
   "Typing :: Typed",
 ]
 dynamic = ["version"]
-dependencies = []
+dependencies = [
+  "streamlit >= 1.32.2",
```
Comment (lsetiawan) [inline pyproject.toml:32]:
```suggestion
  "streamlit>=1.32,<2",
```
Probably don't need the super specific version, also, it's good to pin to the top of the version

File: pyproject.toml [hunk 4]
```diff
@@ -79,7 +82,7 @@ testpaths = [
 
 
 [tool.coverage]
-run.source = ["genetic_forensic_portal"]
+run.source = ["gf_app"]
```
Comment (lsetiawan) [inline pyproject.toml:85]: This probably need to be the full package name no?

File: pyproject.toml [hunk 5]
```diff
@@ -135,13 +138,14 @@ ignore = [
 ]
 isort.required-imports = ["from __future__ import annotations"]
```
Comment (lsetiawan) [inline pyproject.toml:139]:
```suggestion
```
Is a good practice for older python versions, but probably don't need here

File: pyproject.toml [hunk 6]
```diff
@@ -135,13 +138,14 @@ ignore = [
...
 # Uncomment if using a _compat.typing backport
+# TODO change if i ever figure out what this is or that i need it. ?????????
```
Comment (lsetiawan) [inline pyproject.toml:141]: Probably don't need as we are using newer python typing stuff

**Review by lsetiawan — APPROVED** [review]
Few tweaks, otherwise good to go, whenever.

File: .github/workflows/cd.yml (hunk 1 above)
Comment (nikiburggraf) [reply .github/workflows/cd.yml:32]: will revert the commit that does this

File: CONTRIBUTING.md (hunk 2 above)
Comment (nikiburggraf) [reply CONTRIBUTING.md:52]: OOPS thank you

**Pull request merged**